src/tests/unit/services/GenerateVerifiableCredential.test.ts

```diff
@@ -231,6 +251,35 @@ describe("GenerateVerifiableCredential", () => {
 		);
 	});
 
+		it("should return the contra indicator array $contraIndicator for authenticity rejection reason $reason", () => {
```

Comment:
Thanks for changing this to a regular it! We now don't have access to `$reason` and `$contraIndicator` you might see that they are just coming through as the strings when you run the tests, could we please substitute these for the actual values?